packages/logging/src/index.js

```diff
@@ -444,7 +479,46 @@ Logging.prototype.getSinks = function(options, callback) {
  *     this.end();
  *   });
  */
-Logging.prototype.getSinksStream = common.paginator.streamify('getSinks');
+Logging.prototype.getSinksStream = function(options) {
```

Comment:
I've been wondering how to do this, but I haven't figured it out yet. The part I'd like to avoid is the handling of `abort`, because that seems easy to forget-- it'd be nice to not have to remember to do that. The challenge is that the `abort` function comes from the request stream, and:

- we don't immediately make the request, we wait for the user to actually send us some data. This behavior could be removed, and we couldn't instantly make the request, but:

- we'd still have to re-assign the `abort` function, because we are returning a stream which is composed of multiple streams, and not just the request stream directly.